### PR TITLE
Move SRDFWriter to srdfdom, issue #66

### PR DESCRIPTION
**Pull request by velveteenrobot**
Fixes issue #66, merge https://github.com/ros-planning/srdfdom/pull/14 before this.

### CODE REVIEW

**Comment by davetcoleman** [comment]
+1 after srdfdom is merged and released

note: must cherry-pick to J/K

**Comment by rhaschke** [comment]
+1

File: moveit_setup_assistant/package.xml [hunk 1]
```diff
@@ -21,11 +21,13 @@
...
   <build_depend>moveit_ros_visualization</build_depend>
   <build_depend>yaml-cpp</build_depend>
+  <build_depend version_gt="0.3.1">srdfdom</build_depend>
  
   <run_depend>moveit_core</run_depend>
   <run_depend>moveit_ros_planning</run_depend>
   <run_depend>moveit_ros_visualization</run_depend>
   <run_depend>yaml-cpp</run_depend>
   <run_depend>xacro</run_depend>
+  <run_depend version_gt="0.3.1">srdfdom</run_depend>
```
Comment (rhaschke) [inline moveit_setup_assistant/package.xml]: After release of srdfdom, this needs to be increased to the new version, presumably 0.3.2!

Comment (mikeferguson) [inline moveit_setup_assistant/package.xml]: we used a "gt" tag -- which means greater than 0.3.1, so 0.3.2 or higher is already required

Comment (rhaschke) [inline moveit_setup_assistant/package.xml]: But only with the to-be-released srdfdom 0.3.2 the SRDFWriter will be in srdfdom. 0.3.1 will not work anymore.

Comment (v4hn) [inline moveit_setup_assistant/package.xml]: On Wed, Aug 24, 2016 at 02:54:27PM -0700, Robert Haschke wrote:
> > +  <run_depend version_gt="0.3.1">srdfdom</run_depend>
> 
> But only with the to-be-released srdfdom 0.3.2 the SRDFWriter will be in srdfdom. 0.3.1 will not work anymore.

`version_gt` = `>`
`version_gte` = `>='

The patch line above actually requires 0.3.2 or higher.

Comment (gavanderhoorn) [inline moveit_setup_assistant/package.xml]: Maybe using a `version_gte="0.3.2"` would be clearer?

Comment (rhaschke) [inline moveit_setup_assistant/package.xml]: Oh. I was blind-folded yesterday. Of course `version_gt=0.3.1` would be sufficient. Sorry for the noise.

Comment (130s) [inline moveit_setup_assistant/package.xml]: > Maybe using a version_gte="0.3.2" would be clearer?

+0.5. I had the same confusion as @rhaschke. But I guess the version of the depended package (srdfdom in this case) is *technically* out of control from this repo, so I guess the current `version_gt="0.3.1"` is the safest.

Comment (mikeferguson) [inline moveit_setup_assistant/package.xml]: @130s -- that was exactly my thinking, that we don't control the versioning, we could only say "newer than what is currently out"

**Comment by 130s** [comment]
https://github.com/ros-planning/srdfdom/pull/14 is merged and release req is sent for IJK (only J is merged so far).

I https://github.com/ros/rosdistro/pull/12502
J https://github.com/ros/rosdistro/pull/12503
K https://github.com/ros/rosdistro/pull/12505

**Comment by 130s** [comment]
I thought before merging this, we wanted to wait for the debs of `srdfdom` to become available on shadow, to see whether travis would pass or not. If we didn't respect CI much then it would have been valueless. Let's wait for the CIs to pass more strictly in the future ;)

**Comment by rhaschke** [comment]
I did a Travis test on Kinetic beforehand: https://travis-ci.org/rhaschke/moveit/builds/155291875 which succeeded. That's why I thought it's time for merging. Obviously Indigo lacks behind. Is it / Where is it possible to see whether the releases became available?

**Comment by v4hn** [comment]
On Fri, Aug 26, 2016 at 05:18:32AM -0700, Robert Haschke wrote:
> I did a Travis test on Kinetic beforehand: https://travis-ci.org/rhaschke/moveit/builds/155291875 which succeeded.

*Please* give that kind of information *before* you merge.
Otherwise we stumble upon your changes and have no idea why you thought it justified to merge/commit.
Usually you *do* have your reasons :)

**Pull request merged**